shared/interfaces/Tag.d.ts

```diff
@@ -0,0 +1,12 @@
+export interface ITag {
+    _id?: string;
+    text: string;
+}
+export interface PostTagCreateData {
+    postId: string;
+    text: string;
+}
+export interface PostTagDeleteData {
+    postId: string;
+    tagId: string;
+}
```

Comment:
Please follow one convention of naming interface's for all: with -Model suffix; with I- prefix, with -DTO suffix. Do not mix them

Reply:
I fixed this, according Anton Fandul remarks.